reactor-netty-core/src/main/java/reactor/netty/tcp/TcpClient.java

```diff
@@ -240,6 +240,10 @@ public TcpClient metrics(boolean enable) {
 		return super.metrics(enable);
 	}
 
+	public TcpClient metrics(boolean enable, String prefix) {
+		return metrics(enable, () -> new TcpClientConfig.MicrometerTcpClientMetricsRecorder(prefix));
+	}
+
```

Comment:
Is this needed? One can use 
`reactor.netty.transport.Transport#metrics(boolean, java.util.function.Supplier<? extends reactor.netty.channel.ChannelMetricsRecorder>)` with `MicrometerChannelMetricsRecorder`

Reply:
Removed. It just helps the user configure a prefix without the need to know the exact metrics recorder they use.